shared/react-native/buildAndroid.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+IFS=: read -a GOPATH_ARRAY <<< "$GOPATH"
+GOPATH0=${GOPATH_ARRAY[0]}
+
+cd $GOPATH0/src/github.com/keybase/client/shared/react-native/android
```

Comment:
I'm guessing `LOCAL_CLIENT=1` and `LOCAL_KBFS=1` don't matter here since the thing they produce would be in the same place. But thought I'd bring it up anyway in case it does matter.

Reply:
What do those env vars do again?

Reply:
Those are for `rn-gobuild-{ios,android}`, to specify whether we want a fresh checkout or just use what's locally in `$GOPATH` for the gobind build.

Reply:
Yeah I always build with `LOCAL_KBFS=1` (`LOCAL_CLIENT=1` is the default)

Reply:
Ah, yes. Looks like that's only for `rn-gobuild-android` and yeah I think it still ends up in the same place.